lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -433,38 +418,3 @@ def test_html_sidebar_enabled(self, itemfactory_display_name, itemfactory_data,
                 self.assertContains(resp, itemfactory_data)
             else:
                 self.assertNotContains(resp, '<section class="about-sidebar-html">')
-
-
-class CourseAboutTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
-    """
-    Test for unenrolled student tries to access ccx.
-    Note: Only CCX coach can enroll a student in CCX. In sum self-registration not allowed.
-    """
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.course = CourseFactory.create()
-
-    def setUp(self):
-        super().setUp()
-
-        # Create ccx coach account
-        self.coach = coach = AdminFactory.create(password="test")
-        self.client.login(username=coach.username, password="test")
-
-    @override_waffle_flag(COURSE_HOME_USE_LEGACY_FRONTEND, active=True)
-    def test_redirect_to_dashboard_unenrolled_ccx(self):
```

Comment:
This was hitting the outline tab (removed) and expecting a redirect to the dashboard. (You might have thought it was redirecting TO the outline tab and it would be an easy update?)

Reply:
Is anything ever easy?